### PR TITLE
Support of phone numbers with extension in validator

### PR DESCRIPTION
**Pull request by Kukunin**

### CODE REVIEW

**Comment by jarosluv** [comment]
Good work! I just closed my PR in favor of yours.

File: lib/validators/phony_validator.rb [hunk 1]
```diff
@@ -9,7 +9,7 @@ def validate_each(record, attribute, value)
 
     @record = record
     value = PhonyRails.normalize_number(value.dup, default_country_code: normalized_country_code) if normalized_country_code
-    value, = PhonyRails.extract_extension(value)
+    value = PhonyRails.extract_extension(value).first
```
Comment (jarosluv) [inline lib/validators/phony_validator.rb]: This is a good alternative, following Ruby conventions
```value, _ = PhonyRails.extract_extension(value)```

Comment (Kukunin) [inline lib/validators/phony_validator.rb]: Actually rubocop advices `value,  = PhonyRails.extract_extension(value)` instead, which looks confusing

Comment (joost) [inline lib/validators/phony_validator.rb]: Aah.. get it.. bit confusing indeed.

**Pull request merged**